src/app/Classes/Template/Builders/Structure.php

```diff
@@ -30,7 +30,7 @@ private function readPath()
                 ?? config('enso.datatable.dataRouteSuffix')
         );
 
-        $this->template->readPath = route($route, [], false);
+        $this->template->readPath = route($route, [], config('enso.datatable.dataPathAbsolute', false));
```

Comment:
- [ ] let's rename the option to `absolutePath'
- [ ] add this option to the config file after the `method` options and document it
- [ ] update `Template/Builders/Buttons.php` and `Tests/Datatable.php` too
- [ ] update the [docs](https://docs.laravel-enso.com/packages/vue-datatable.html)